app/src/main/scala/org/alephium/explorer/persistence/schema/HashrateSchema.scala

```diff
@@ -36,13 +35,13 @@ trait HashrateSchema extends CustomTypes {
 
     def pk: PrimaryKey = primaryKey("hashrates_pk", (timestamp, intervalType))
 
+    def timestampIdx: Index    = index("hashrates_timestamp_idx", timestamp)
+    def intervalTypeIdx: Index = index("hashrates_interval_type_idx", intervalType)
```

Comment:
We need to create partial index for timestamp I believe, i.e. `CREATE INDEX hashrates_timestamp_idx on timestamp where intervalType = 0`.

@simerplaha Do you think my comment makes sense? I remember you have built partial indexes for other tables already

Reply:
Yep that makes sense. Just tried looking if I had benchmarks documented anywhere for adding partial index (`main_chain = true`) to `block_headers` table ([PR:131](https://github.com/alephium/explorer-backend/pull/131)) but I think I lost it. My bad :(. Couldn't find the code for the benchmark either. We only benchmark with or without the index. 

Benchmarking `hashrates_timestamp_idx ` index would tell us the truth. Please let me know if you'd like me to run the benchmark?

Reply:
if you have the courage, yes please that would be super kind :heart:

Reply:
we will actually do it in another task: https://github.com/alephium/explorer-backend/issues/158